app/models/mixins/relationship_mixin.rb

```diff
@@ -306,12 +378,20 @@ def sibling_count(*args)
 
   # Returns true if the record's parent has more than one child
   def has_siblings?(*_args)
```

Comment:
wondering why we have even have `*_args`

Reply:
yeah i'm not sure why we did this but it felt like a bunch of changes without worrying about it